Framework/Algorithms/src/CreateWorkspace.cpp

```diff
@@ -144,16 +161,15 @@ void CreateWorkspace::exec() {
   }
 
   // Create the OutputWorkspace
+  auto storageMode = Parallel::fromString(getProperty("ParallelStorageMode"));
```

Comment:
Actually, I probably want to remove the `#ifdef` on the property -- otherwise I cannot run the new unit tests in a non-MPI build